tests/testPsfAttributes.py

```diff
@@ -36,16 +36,13 @@
 
 import lsst.afw.geom as afwGeom
 import lsst.meas.algorithms as measAlg
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
 try:
     type(verbose)
     import lsst.afw.display.ds9 as ds9
 except NameError:
     verbose = 0
-    logging.Trace.setVerbosity("meas.algorithms.Interp", verbose)
-    logging.Trace.setVerbosity("afw.detection.Psf", verbose)
```

Comment:
I think it'd be good to do the same thing here that you do in testPsfDetermination: set logging to INFO or DEBUG or TRACE with comment on how to increase log level.